7Cav-Alive.Altis/initServer.sqf

```diff
@@ -1,7 +1,7 @@
 diag_log "initServer start";
+missionNamespace setVariable["LOYALTY_CIPHER", CIPHERSECRETKEY, true]; //variable init for loyalty cipher
+missionNamespace setVariable["endMissionNumber",0]; //Variable init for missionEnd scripts located in scripts/missionEnd
```

Comment:
You have this in both `initServer.sqf` and `fn_serverPreInit.sqf`

Reply:
I shouldn't, it should have been changed to not do that.

Reply:
This is still an issue